### DISCUSSION TITLE
add/update cluster_vip for the Type A

### DISCUSSION THREAD

**Discussion by argabi** [autobase-tech/autobase#175] [Q&A] [answered]
Dear @vitabaks,

I have a production cluster [Type A], when I deployed it, I had not specified the `cluster_vip`.
Now I want to add a VIP for my F5 LoadBalancer which is in front of my cluster nodes. (F5 front of  3 nodes)

My question is how can I Add/Update the `cluster_vip` managed by F5. please step by step. 
It is a production environment and has data.

Thank you.

**Comment by vitabaks** [comment, marked as answer]
I am not familiar with F5 LoadBalancer, but to add an VIP address for `haproxy`, you can do the following:

1. specify `cluster_vip` in [vars](https://github.com/vitabaks/postgresql_cluster/tree/master/vars)/main.yml
2. add haproxy nodes [balancers] in the `inventory` file 
3. run `ansible-playbook balancers.yml --tag keepalived`

**Reply by vitabaks** [reply]
Please test it in your test enviroment before using in a production.

**Reply by argabi** [reply]
Thank you for the fast replay.
I do not have a test environment for DBs.
However, I'll try these steps.

Again thank you.
